src/App.css

```diff
@@ -1,42 +1,63 @@
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+    max-width: 1280px;
+    margin: 0 auto;
+    padding: 2rem;
+    text-align: center;
 }
 
 .logo {
```

Comment:
The the `logo` class not get removed in the recent template? We might be able to clean some of these away.

We should at least should be avoiding `will-change` except as a last resort.